edx-platform/pearson-vue-theme/lms/static/partials/lms/theme/_courses.scss

```diff
@@ -1,604 +1,767 @@
-      }
-    }
+li.card.list-card.team-card,
+li.card.square-card.topic-card.has-pennant {
+  border: 1px solid $secondary;
+}
 
-    .facet-option {
-      @include float(left);
-      @include transition(all $tmg-f2 ease-out 0s);
-      @include clearfix();
+div.page-header-main nav.breadcrumbs a.nav-item {
+  color: $primary;
+}
 
-      @extend %t-action3;
-      @extend %text-truncated;
+@media (max-width: 767px) {
+  .view-teams  {
+    div.team-profile div.page-content-main,
+    div.team-profile div.page-content-secondary {
+      width: 100%;
+    }
 
-      opacity: 1;
-      border-radius: 0;
-      padding: ($baseline/4) ($baseline/2);
+    li.card.list-card div.card-actions,
+    li.card.square-card.topic-card.has-pennant {
       width: 100%;
-      color: $facet-text-color;
-
-      .count {
-        @include right($baseline*0.6);
-        @include text-align(right);
-
-        box-sizing: border-box;
-
-        @include transition(all 0.2s ease-out);
-
-        position: absolute;
-        width: ($baseline*2);
-      }
-
-      //STATE: hover, focus
-      &:hover,
-      &:focus {
-        background: $facet-background-color;
-        color: $white;
-        text-decoration: none;
-
-        .count {
-          color: $white;
-        }
-      }
-
-      //STATE: selected (hover, focus)
-      &.selected,
-      &.selected:hover,
-      &.selected:focus {
-        background: $gray-l4;
-        color: $facet-text-color;
-
-        .count {
-          color: theme-color("dark");
-
-          .count-number {
-            display: none;
-          }
-        }
-
-        .count::before {
-          @include left($baseline*0.75);
-
-          position: absolute;
-          width: ($baseline*2);
-          color: $gray-l1;
-          font-family: FontAwesome;
-          text-align: center;
-          content: '\f00d'; // fa-times
-        }
-      }
     }
+  }
 
-    .toggle {
-      @include clearfix();
+}
 
-      button {
-        @extend %t-icon6;
-        @extend %t-strong;
+div.gradebook-wrapper section.gradebook-content
+table.student-table td a {
+  color: $primary-black;
+}
 
-        padding: ($baseline/4) ($baseline/2);
-        color: $facet-background-color;
-      }
-    }
+div.gradebook-wrapper section.gradebook-content
+table.grade-table td {
+  color: $primary-black;
+}
+
+div.gradebook-wrapper section.gradebook-content .student-search-field {
+  color: $primary-font-color;
+}
+
+.ui-dialog-titlebar.ui-widget-header.ui-corner-all.ui-helper-clearfix {
+  background: $primary;
+  border: 1px solid;
+}
+
+.view-teams div.wrapper-msg {
+  background: $inverse;
+  border-top: 3px solid $primary;
+  .right-floater button {
+    color: $primary;
   }
 }
 
-// +All Other Styles
-// ====================
-.find-courses,
-.university-profile {
+.wrapper-prompt .prompt div.nav-actions {
+  background: $secondary;
+}
+
+span.ui-button-icon-primary.ui-icon.ui-icon-closethick {
+  margin: -9px;
+}
+
+li.errors {
+  color: red;
```

Comment:
Is there already a variable for this value? @luismorenolopera

Reply:
I think it would be redundant